src/main/java/io/nats/client/impl/FileAuthHandler.java

```diff
@@ -13,16 +13,16 @@
 
 package io.nats.client.impl;
 
+import io.nats.client.AuthHandler;
+import io.nats.nkey.NKey;
```

Comment:
Safe to use change since NKey is internal to class and not returned in any public api of the class.